### DISCUSSION TITLE
TSX and TS are different

### DISCUSSION THREAD

**Discussion by AviralCoder** [github-linguist/linguist#5609] [Classification] [answered]
In linguist, .tsx aren't taken as typescript files, only .ts are taken as typescript files..why?

**Comment by lildude** [comment, marked as answer]
I guess you didn't [search the Issues for TSX](https://github.com/github/linguist/issues?q=is%3Aissue+tsx+is%3Aclosed) 😉 

See my response at https://github.com/github/linguist/issues/5485#issuecomment-888041136

**Reply by AviralCoder** [reply]
Oh thanks.
